.github/workflows/CI.yml

```diff
@@ -70,8 +70,8 @@ jobs:
       - run: |
           julia --project=docs -e '
             using Pkg
-            Pkg.add(url="https://github.com/JuliaAtoms/AtomicLevels.jl.git")
-            Pkg.develop(PackageSpec(path=pwd()))
+            #Pkg.add(url="https://github.com/JuliaAtoms/AtomicLevels.jl.git")
```

Comment:
I just added AtomicLevels as a dependency to `docs/Project.toml`. Can these be removed?

Reply:
Removed these lines.